### ISSUE TITLE
🐛 Bug Report: [Docs] Permission Diagram Hard to Read

### ISSUE THREAD

**Issue by awanlin** [backstage/backstage#19558]
### 📜 Description

The permission digram under the "How Does It Work" heading is really hard to read: https://backstage.io/docs/permissions/overview#how-does-it-work

### 👍 Expected behavior

The digram should be easy to read

### 👎 Actual Behavior with Screenshots

![image](https://github.com/backstage/backstage/assets/67169551/b15af16b-b6fc-433a-ad95-04dbdfc46804)

### 👟 Reproduction steps

1. Go to: https://backstage.io/docs/permissions/overview#how-does-it-work

Note: the digram is hard to read

### 📃 Provide the context for the Bug.

Was just sharing these docs and noticed that this diagram is hard to read

### 🖥️ Your Environment

No really an environment issue

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

No, but I'm happy to collaborate on a PR with someone else

**Comment by awanlin** [comment]
This is the file for the diagram being used: https://github.com/backstage/backstage/blob/master/docs/assets/permissions/permission-framework-overview.drawio.svg

Maybe if it didn't have a black background it would be much better?

**Comment by vinzscam** [comment]
yeah makes sense.

> Maybe if it didn't have a black background it would be much better?

Maybe we should toggle the colors of the diagram to match with the dark theme? 🤔

**Comment by awanlin** [comment]
Yeah, matching the theme would be nice but for now I've just made it white as that was an easy quick fix: https://github.com/backstage/backstage/pull/19623
